net.sourceforge.jmol.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=Jmol
+Exec=jmol.sh
```

Comment:
there should also be a `Comment`, `GenericName`, and `Categories`

https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html